roles/csv2-ci/vars/csv2-ci-vars.yaml

```diff
@@ -2,10 +2,10 @@
 timezone: America/Vancouver
 
 # Admin email for root mail forwarding
-admin_email: root@localhost
```

Comment:
this can stay as root@localhost

Reply:
changed in [1b62142](https://github.com/hep-gc/uvic-heprc-ansible-playbooks/pull/14/commits/1b62142613004efbe73b44a7be7e5d7e9cdc5064)